x/axelarnet/module.go

```diff
@@ -305,19 +300,36 @@ func (am AppModule) OnTimeoutPacket(
 		return err
 	}
 
-	t, err := types.PacketToTransfer(packet)
-	if err != nil {
-		return err
-	}
-	// requeue transfer
-	err = am.keeper.EnqueueTransfer(ctx, t)
-	if err != nil {
-		return err
-	}
-	return nil
+	return setTransferFailed(ctx, am.keeper, packet.SourcePort, packet.SourceChannel, packet.Sequence)
 }
 
 // NegotiateAppVersion implements the IBCModule interface
 func (am AppModule) NegotiateAppVersion(ctx sdk.Context, order channeltypes.Order, connectionID string, portID string, counterparty channeltypes.Counterparty, proposedVersion string) (version string, err error) {
 	return am.transferModule.NegotiateAppVersion(ctx, order, connectionID, portID, counterparty, proposedVersion)
 }
+
+// returns true if mapping exits
+func getSeqIDMapping(ctx sdk.Context, k keeper.Keeper, portID, channelID string, seq uint64) (nexus.TransferID, bool) {
+	k.DeleteSeqIDMapping(ctx, portID, channelID, seq)
+
+	return k.GetSeqIDMapping(ctx, portID, channelID, seq)
+}
+
+func setTransferFailed(ctx sdk.Context, k keeper.Keeper, portID, channelID string, seq uint64) error {
+	transferID, ok := getSeqIDMapping(ctx, k, portID, channelID, seq)
```

Comment:
Add a comment about why this mapping won't legitimately exists for user transfers